.gitignore

```diff
@@ -115,3 +115,6 @@ a.md
 
 # Ignore a gunicorn config file
 gunicorn.conf.py
+
+# GraphQL generated schema
+mlflow/server/graphql/do_not_modify_autogenerated_schema.gql
```

Comment:
what uses this file?

Reply:
It is an interemediate file that is used by `graphql-codegen.ts`, you can see in that file it is referencing this .gql file

Reply:
got it, makes sense to ignore